win32/tests/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ if(${run_unittests})
     endif()
 
     build_test_folder(gballoc_hl_passthrough_ut)
+    build_test_folder(job_object_helper_ut)

```

Comment:
>job_object_helper_ut [](http://example.com/codeflow?start=22&length=20)

I believe we need some job_object_int tests.

I would like to see if the API `job_object_helper_limit_resources` can be called twice, and if so, what's the effects.

Specifically, can the memory/CPU be un-restricted or only more-and-more restricted?

Also we can check that "it really works" with int tests,, like restrict memory to 1GB, try to allocate 1GB => fail. 

"unrestrict" memory, try to allocate 1GB=> success.

And so on.

Reply:
Job Object nesting and chains are... interesting.

Reply:
Anyway, I would be waiting because reasons, but can only sign off.

Reply:
will create a task